extensions/amp-google-read-aloud-player/0.1/test/validator-amp-google-read-aloud-player.out

```diff
@@ -0,0 +1,127 @@
+FAIL
+|  <!--
+|    Test Description:
+|    Tests for amp-google-read-aloud-player tag.
+|  -->
+|  <!doctype html>
+|  <html ⚡>
+|
+|  <head>
+|    <meta charset="utf-8">
+|    <title>amp-google-read-aloud-player example</title>
+|    <link rel="canonical" href="amps.html">
+|    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+|    <style amp-boilerplate>
+|      body {
+|        -webkit-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+|        -moz-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+|        -ms-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+|        animation: -amp-start 8s steps(1, end) 0s 1 normal both
+|      }
+|
+|      @-webkit-keyframes -amp-start {
+|        from {
+|          visibility: hidden
+|        }
+|
+|        to {
+|          visibility: visible
+|        }
+|      }
+|
+|      @-moz-keyframes -amp-start {
+|        from {
+|          visibility: hidden
+|        }
+|
+|        to {
+|          visibility: visible
+|        }
+|      }
+|
+|      @-ms-keyframes -amp-start {
+|        from {
+|          visibility: hidden
+|        }
+|
+|        to {
+|          visibility: visible
+|        }
+|      }
+|
+|      @-o-keyframes -amp-start {
+|        from {
+|          visibility: hidden
+|        }
+|
+|        to {
+|          visibility: visible
+|        }
+|      }
+|
+|      @keyframes -amp-start {
+|        from {
+|          visibility: hidden
+|        }
+|
+|        to {
+|          visibility: visible
+|        }
+|      }
+|    </style><noscript>
+|      <style amp-boilerplate>
+|        body {
+|          -webkit-animation: none;
+|          -moz-animation: none;
+|          -ms-animation: none;
+|          animation: none
+|        }
+|      </style>
+|    </noscript>
+|    <script async custom-element="amp-google-read-aloud-player" src="https://cdn.ampproject.org/v0/amp-google-read-aloud-player-0.1.js"></script>
+|    <script async src="https://cdn.ampproject.org/v0.js"></script>
+|  </head>
+|
+|  <body>
+|    <!-- Valid -->
+|    <amp-google-read-aloud-player height="66" data-api-key="YOUR_API_KEY" data-tracking-ids="UA-168915890-1" data-voice="en.hol-wavenet">
+>>   ^~~~~~~~~
+amp-google-read-aloud-player/0.1/test/validator-amp-google-read-aloud-player.html:86:2 The tag 'amp-google-read-aloud-player' is disallowed.
```

Comment:
This is marked as `<!-- Valid -->`, but it's failing. There's likely a validation config issue.

Reply:
For some reason the validator does not recognize the new extension. The validator was removed from this PR and will be added in a different one.